src/Controller/AdClickCommandController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Drupal\adclickcommand\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\KeyValueStore\KeyValueExpirableFactoryInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Entity\Query\QueryFactory;
+
+/**
+ * Returns responses for PuSH module routes.
+ */
+class AdClickCommandController extends ControllerBase implements ContainerInjectionInterface {
+
+  /**
+   * The key value expirable factory.
+   *
+   * @var \Drupal\Core\KeyValueStore\KeyValueExpirableFactoryInterface
+   */
+  protected $keyValueExpireFactory;
+
+  /**
+   * The entity query factory.
+   *
+   * @var \Drupal\Core\Entity\Query\QueryFactory
+   */
+  protected $entityQuery;
```

Comment:
I added `$entityQuery` as a property in this class to be the entity query factory that gets injected from the container.